src/quo/components/links/internal_link_card/user/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo.components.links.internal-link-card.user.view
+  (:require
+    [quo.components.icon :as icon]
+    [quo.components.links.internal-link-card.user.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [react-native.linear-gradient :as linear-gradient]))
+
+(defn- subtitle-comp
+  [subtitle emojis]
+  [rn/view
+   [text/text
+    {:size                :paragraph-2
+     :number-of-lines     2
+     :accessibility-label :subtitle
+     :style               {:margin-bottom 12}} subtitle]
+   [rn/view {:flex-direction :row}
+    (doall
```

Comment:
I'm not seeing side-effects to build up this hiccup, so why `doall`?

Reply:
Removed, Previously Metro was throwing a warning without `doall`.